### PR TITLE
Added SidebarCheckbox instruction, JobEmploymentType instruction.

### PR DESCRIPTION
**Pull request by hansjovis**
## Summary

This PR can be summarized in the following changelog entry:

* [schema-blocks] Added `SidebarCheckbox` instruction.
* [schema-blocks] Added `JobEmploymentType` instruction.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Needs to be tested in tandem with [this PR on Premium](https://github.com/Yoast/wordpress-seo-premium/pull/3174). Please follow its test instructions.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * N/A: only affects the schema blocks functionality, which is behind a feature flag.

## UI changes
* [x] This PR changes the UI in the plugin. I have added the 'UI change' label to this PR.

## Quality assurance

* [x] I have tested this code to the best of my abilities
* [ ] I have added unittests to verify the code works as intended

Fixes #

### CODE REVIEW

File: packages/schema-blocks/src/core/blocks/BlockDefinition.ts [hunk 1]
```diff
@@ -46,8 +47,9 @@ export default class BlockDefinition extends Definition {
 			.filter( e => e !== null );
 		if ( sidebarElements.length > 0 ) {
 			// Need to add `children` on the `props` as well, because of the type definition of `InspectorControls.Props`.
-			const sidebar = createElement( InspectorControls, { key: "sidebar", children: sidebarElements }, sidebarElements );
-			elements.unshift( sidebar );
+			const sidebar = createElement( PanelBody, { children: sidebarElements }, sidebarElements );
+			const sidebarContainer = createElement( InspectorControls, { key: "sidebar", children: [ sidebar ] }, [ sidebar ] );
+			elements.unshift( sidebarContainer );
```
Comment (hansjovis) [inline packages/schema-blocks/src/core/blocks/BlockDefinition.ts:52]: Note: I added this wrapper component to fix the padding of the sidebar elements we add.

File: packages/schema-blocks/src/core/blocks/BlockDefinition.ts [hunk 2]
```diff
@@ -46,8 +47,9 @@ export default class BlockDefinition extends Definition {
 			.filter( e => e !== null );
 		if ( sidebarElements.length > 0 ) {
 			// Need to add `children` on the `props` as well, because of the type definition of `InspectorControls.Props`.
-			const sidebar = createElement( InspectorControls, { key: "sidebar", children: sidebarElements }, sidebarElements );
-			elements.unshift( sidebar );
+			const sidebar = createElement( PanelBody, { children: sidebarElements }, sidebarElements );
```
Comment (johannadevos) [inline packages/schema-blocks/src/core/blocks/BlockDefinition.ts:50]: I think you need to add a `key`.

File: packages/schema-blocks/src/instructions/schema/JobEmploymentType.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	 * Renders schema.
+	 *
+	 * @param block The block.
+	 *
+	 * @returns The schema.
+	 */
+	render( block: BlockInstance ): SchemaArray | string {
+		const employmentType = block.attributes.employmentType;
+		const { isVolunteer, isIntern } = block.attributes;
+
```
Comment (johannadevos) [inline packages/schema-blocks/src/instructions/schema/JobEmploymentType.ts:23]:
```suggestion
```

**Review by andizer — APPROVED** [review]
CR done 👍

File: packages/schema-blocks/src/core/blocks/BlockDefinition.ts (hunk 2 above)
Comment (increddibelly) [inline packages/schema-blocks/src/core/blocks/BlockDefinition.ts:50]:
```suggestion
			const sidebar = createElement( PanelBody, { key: "sidebarPanelBody", children: sidebarElements }, sidebarElements );
```

**Pull request merged**